### ISSUE TITLE
Add links to artist's albums/merchandise when using Bandcamp

### ISSUE THREAD

**Issue by nukeop** [nukeop/nuclear#820]
This would mesh with: https://github.com/nukeop/nuclear/issues/131

The bandcamp scraper gives us links to relevant pages. It would be a nice way to support the artists if the links were displayed on artist's and album's pages in a prominent place (up to implementer where exactly). In the future we could promote interested artists by displaying their bandcamp links somewhere in the dashboard.

**Comment by Yahyakiani** [comment]
@nukeop Can I have this issue? First timer here thanks

**Comment by nukeop** [comment]
Sure, I'll assign it to you.

**Comment by 5usana** [comment]
Hello! If this is still an open issue i'd love a shot at tackling it. May I?

**Comment by nukeop** [comment]
> Hello! If this is still an open issue i'd love a shot at tackling it. May I?

Sure, no problem. This still needs some improvements in the bandcamp scraper to provide links to merchandise.

**Comment by jreiser1** [comment]
Hello, I've got a working prototype for this issue. Can I be assigned to it? I'm hoping to open a pull request in the next day or so. 

Right now I've implemented an Artist Info Button on the ArtistView header that will open up the artist's Bandcamp/iTunes/Discogs/Audius/Musicbrainz page depending on which metadata provider the user has selected. Audius, Discogs, and MusicBrainz will currently only open a search page with the artist's name provided, but I think that's a good start at least. Bandcamp and iTunes will open directly to the artist's page.

![image](https://user-images.githubusercontent.com/47801590/141892845-52040946-56c9-4809-91ae-c46e584e4eb8.png)
